config/config.go

```diff
@@ -2118,3 +2118,47 @@ func ParseRemoteState(ctx context.Context, l log.Logger, opts *options.Terragrun
 
 	return cfg.GetRemoteState(l, opts)
 }
+
+func CleanStacks(_ context.Context, l log.Logger, opts *options.TerragruntOptions) error {
+	if opts.TerraformCommand == tf.CommandNameDestroy {
```

Comment:
I'm assuming you tested this, but do you mind documenting why destroys shouldn't be cleaned?